tools/lambda-promtail/lambda-promtail/promtail.go

```diff
@@ -103,6 +116,18 @@ func (b *batch) createPushRequest() (*logproto.PushRequest, int) {
 	return &req, entriesCount
 }
 
+func (b *batch) flushBatch(ctx context.Context) error {
+	fmt.Println("flusing batch")
```

Comment:
might be useful to include some additional information here, we could include the batch size in the log line?

Reply:
this was commited by mistake, will remove it like Owen suggested